jenkins/automation/build_retention/build_retention.groovy

```diff
@@ -0,0 +1,52 @@
+pipeline {
+
+	agent {
+		node {
+				label 'build-retention'
+		}
+	}
+	parameters {
+
+		string(name: 'BUILD', defaultValue: '"561","531","2750"', description: 'Build Number')
+		string(name: 'BRANCH', defaultValue: '"main","custom-ci"', description: 'Branch Name')
+		string(name: 'OS', defaultValue: '"centos-7.8.2003","centos-7.9.2009","rhel-7.7.1908"', description: 'OS Version')
+
+
+    }
+	
+	stages {
+		
+		stage('Checkout Script') {
+            steps {             
+                script {
+                    checkout([$class: 'GitSCM', branches: [[name: '*/Build-Retention']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'cortx-admin-github', url: 'https://github.com/nitisdev/cortx-re']]])   
```

Comment:
Change repo and branch to main once testing is done

Reply:
Done